Form/FormMapper.php

```diff
@@ -305,4 +308,20 @@ protected function setTabs(array $tabs)
     {
         $this->admin->setFormTabs($tabs);
     }
+
+
+    /**
+     * Symfony default form class sadly can't handle
+     * form element with dots in its name (when data
+     * get bound, the default dataMapper is a PropertyPathMapper).
+     * So use this trick to avoid any issue.
+     *
+     * @param string $fieldName
+     *
+     * @return string
+     */
+    public function sanitizeFieldName($fieldName)
+    {
+        return str_replace(['__', '.'], ['____', '__'], $fieldName);
```

Comment:
You can't use short array syntax right now, we are still trying to stay php 5.3-compatible